x/stake/handler.go

```diff
@@ -462,6 +477,9 @@ func handleMsgBeginUnbonding(ctx sdk.Context, msg types.MsgBeginUnbonding, k kee
 }
 
 func handleMsgRedelegate(ctx sdk.Context, msg types.MsgRedelegate, k keeper.Keeper) sdk.Result {
+	if sdk.IsUpgrade(sdk.BCFusionSecondHardFork) {
```

Comment:
`handleMsgDelegate` should also be disabled

Reply:
delegate should be disable since the second hardfork.

Reply:
`MsgSideChainDelegate` has been disabled since the second hardfork.